AGGREGATE.md

```diff
@@ -187,6 +187,26 @@ The `payload` will need to contain all the information needed for the aggregatio
 
 The payload should be encrypted via [HPKE](https://datatracker.ietf.org/doc/draft-irtf-cfrg-hpke/), to public keys specified by the processing origins at some well-known address  `/.well-known/aggregation-service/keys.json` that the browser can fetch. Note that we are avoiding using the `attribution-reporting` namespace because many APIs may want to use this infrastructure beyond attribution reporting.
 
+The processing origin will respond with a set of keys which will be stored according to standard HTTP caching rules, i.e. using Cache-Control headers to dictate how long to store the keys for (e.g. following the [freshness lifetime](https://datatracker.ietf.org/doc/html/rfc7234#section-4.2)). The browser could enforce maximum/minimum lifetimes of stored keys to encourage faster key rotation and/or mitigate bandwidth usage. The scheme of the JSON encoded public keys is as follows:
```

Comment:
Could we add a bit more color about how these keys gets used to the start of this paragraph.

The browser will encrypt payloads just before the report is sent by requesting public keys for the processing origin. `keys.json` will be fetched with an un-credentialed request.

Let's also add a note below the schema like:

Note: The browser may need some mechanism to ensure that the same set of keys are delivered to different users.

Reply:
Thanks John. Updated per your suggestion.